src/main/scala/edu/ie3/mobsim/model/TripSimulation.scala

```diff
@@ -230,7 +230,7 @@ object TripSimulation extends LazyLogging {
                 s"${ev.getId} drives to a charging hub of type chargingHubTown."
                 // )
                 makeTripToChargingHub(
-                  "chargingHubTown",
+                  PoiTypeDictionary("chargingHubTown"),
```

Comment:
Please use the enum value directly:
```suggestion
                  PoiTypeDictionary.CHARGING_HUB_TOWN,
```